src/dialogs.ycp

```diff
@@ -77,20 +77,20 @@ term EditDialogContents()
 					    `VBox(
 						  // combo box label: enable FCoE (yes/no)
 						  `ComboBox( `id(`fcoe), _("&FCoE Enable"),
-							     [`item(`id("yes"), "yes" ),
-							      `item(`id("no"), "no", true ) ]
+							     [`item(`id("yes"), _("Yes") ),
```

Comment:
`String::YesNo` is not available in SLE 11 SP4. https://github.com/yast/yast-yast2/blob/Code-11-SP4/library/types/src/String.ycp